dev-util/gammaray/gammaray-2.9.0.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+QT_MINIMAL="5.9"
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay \
+		https://www.kdab.com/?s=gammaray"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
```

Comment:
this being the only `LICENSE`, but then having `USE=gpl-only-features` is a bit strange. Nevertheless in `3rdparty` subdirectory we also find files under `BSD` license regardless of that setting. Imo we can drop this flag and fix `LICENSE` instead.

Reply:
[This](https://github.com/KDAB/GammaRay/blob/master/3rdparty/backward-cpp/LICENSE.txt) license is MIT and [this](https://github.com/KDAB/GammaRay/blob/master/3rdparty/StackWalker/LICENSE) is BSD. How can someone choose?